src/robusta/core/playbooks/playbooks_event_handler.py

```diff
@@ -1,13 +1,13 @@
-from abc import abstractmethod
-from typing import Optional, Dict, Any, List, Protocol
+from abc import ABC, abstractmethod
+from typing import Any, Dict, List, Optional
 
-from ...model.playbook_action import PlaybookAction
-from ...core.model.events import ExecutionBaseEvent
-from ...core.playbooks.base_trigger import TriggerEvent
-from ...runner.telemetry import Telemetry
+from robusta.core.model.events import ExecutionBaseEvent
+from robusta.core.playbooks.base_trigger import TriggerEvent
+from robusta.model.playbook_action import PlaybookAction
+from robusta.runner.telemetry import Telemetry
 
 
-class PlaybooksEventHandler(Protocol):
+class PlaybooksEventHandler(ABC):
```

Comment:
why was this changed from protocol?

Reply:
Protocol is a thing used for typechecking the ducktyping and does nothing in runtime, when here abstractmethods were used, that require ABC to work properly, so I just considered it as just an error.
In any case I except no runtime difference from this change